test/mpi/dtpools/configure.ac

```diff
@@ -49,11 +49,11 @@ AC_ARG_VAR([DTP_DATATYPE_FILE],
            [used to pass MPI datatypes to the DTPools framework (default: "basictypelist.txt")])
 
 # we must have a valid file containing MPI datatypes for framework initialization
-DTP_DATATYPE_FILE=${DTP_DATATYPE_FILE:-$srcdir/basictypelist.txt}
+DTP_DATATYPE_FILE=${DTP_DATATYPE_FILE:-$srcdir/../basictypelist.txt}
```

Comment:
This kind of breaks the independence of DTPools if it requires access to files outside its directory.

Reply:
This is why I was copying the file inside DTPools directory at the beginning. Nevertheless in stand alone mode the user should pass the file to the framework because the framework does not do any assumption on the types supported.

Reply:
Is this then also a candidate for being pushed into the environment by `test/mpi/configure` like `DTP_NUM_OBJS`?

Reply:
No I think for now it's better to leave this  out. Currently this variable is substituted by `test/mpi/dtpools/configure` in `test/mpi/dtpools/src/Makefile.am`. We can think about how to make this cleaner for next release maybe?

Reply:
Alright. I'm going to go ahread with #3085 and we can collaborate on this PR more going forward.

Reply:
Sounds good, thanks